### PR TITLE
Add an index html page

### PR DESCRIPTION
**Pull request by vujita**
Adding an index html page, we probably need to figure a folder structure for static assets, maybe?

--html
--js
--css
--images

### CODE REVIEW

**Comment by vujita** [comment]
If we use webpack, we probably want to keep all of it in a more relative fashion based on components

File: src/app.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ function createWindow() {
   mainWindow = new BrowserWindow({width: 800, height: 600});
 
   // and load the index.html of the app.
-  mainWindow.loadURL(`file://${__dirname}/../html/index.html`);
+  const mainWindowHtml = `file://${__dirname}/../html/index.html`;
+  console.log('How do we want to load this?', mainWindowHtml);
+  mainWindow.loadURL(mainWindowHtml);
```
Comment (alanhamlett) [inline src/app.ts]: The HTML page loaded here should just load the React app.

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (alanhamlett) [inline html/index.html]: Let's not have an `html` folder with just one file. Instead, just put `index.html` up one folder.

Comment (vujita) [inline html/index.html]: sure that works

**Pull request closed without merging**